driver-core/src/main/java/com/datastax/driver/core/Cluster.java

```diff
@@ -1162,6 +1162,8 @@ private Manager(String clusterName, List<InetSocketAddress> contactPoints, Confi
 
             this.metrics = configuration.getMetricsOptions() == null ? null : new Metrics(this);
             this.listeners = new CopyOnWriteArraySet<Host.StateListener>(listeners);
+
+            this.scheduledTasksExecutor.scheduleWithFixedDelay(new CleanupIdleConnectionsTask(), 10, 10, TimeUnit.SECONDS);
```

Comment:
That sounds acceptable to me,  thanks for the explanation.  I agree with minimizing the number of threads.